src/Microsoft.ServiceFabric.Actors/Runtime/ActorConcurrencyLock.cs

```diff
@@ -93,37 +93,37 @@ internal int Test_CurrentCount
             await this.reentrantLock.WaitAsync(cancellationToken);
             try
             {
-                // A new logical call context is appended to every outgoing method call.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.